Client/Experiments/Messaging/GleanPlumbMessageStore.swift

```diff
@@ -32,12 +32,7 @@ class GleanPlumbMessageStore: GleanPlumbMessagingStoreProtocol {
 
     private let decoder = JSONDecoder()
     private let encoder = JSONEncoder()
-
-    private let messagingUtility: GleanPlumbMessageUtility
-
-    init(messagingUtility: GleanPlumbMessageUtility = GleanPlumbMessageUtility()) {
-        self.messagingUtility = messagingUtility
-    }
+    static let rootKey = "GleanPlumb.Messages."
```

Comment:
I understand we'd want to stay away from hard coded strings, but I'm hesitant about this one. 

One option is that we can forgive that hard coded string and leave it be. It's used in one place only, so I'd lean towards this for now.

Although it's not the same, another option is to go with private (set) so the tests can see it, and leave a comment saying this one's a Unit test addition and not meant to be changed.

Reply:
I can add a comment but we can't add private (set) to read-only properties.